src/main/scala/rocketchip/Periphery.scala

```diff
@@ -17,6 +17,11 @@ import rocket.XLen
 import scala.math.max
 import coreplex._
 
+case object MIFAddrBits extends Field[Int]
+case object MIFDataBits extends Field[Int]
+case object MIFTagBits extends Field[Int]
+case object MIFDataBeats extends Field[Int]
+
```

Comment:
I think their main use is to change MIFDataBits to fit whatever interface width you have on the outside.

Probably the best way to do this is to change them to be ExtMemWidth and ExtMMIOWidth.